src/molecules/LiistPlaceCardNew.js

```diff
@@ -210,31 +210,55 @@ export class LiistPlaceCardNew extends LitElement {
     super();
   }
 
+  // after the component is first rendered in its initial state (displayed to user)
+  // this function gets executed
+  firstUpdated() {
+    this.setImage();
+  }
+
+  setImage() {
+    const placeThumbnail = this.shadowRoot.querySelector("img.place-thumbnail");
+    const loadingPlaceholder = this.shadowRoot.querySelector(".loader-placeholder");
+    placeThumbnail.onload = function () {
+      placeThumbnail.classList.remove("hide");
+      placeThumbnail.classList.add("show");
+      loadingPlaceholder.classList.remove("show");
+      loadingPlaceholder.classList.add("hide");
+    }
+    placeThumbnail.src = this.image;
+  }
+
+  isOpen() {
+    return this.status === "OPEN";
+  }
+
+  isClosed() {
+    return this.status === "CLOSED";
+  }
+
   replaceData(obj) {
-    this.name = obj.name;
-    this.address = obj.address;
-    this.itemId = obj.itemId;
-    this.status = obj.status;
-    if (obj.image !== null && obj.image !== undefined) {
-      this.image = obj.image;
-    } else {
-      this.image = "https://firebasestorage.googleapis.com/v0/b/liist-prod.appspot.com/o/placeNotFound.png?alt=media&token=a3f6cf7d-876c-45eb-a69c-1d20723c0db4";
+    this.name = obj.name || this.name;
+    this.address = obj.address || this.address;
+    this.itemId = obj.itemId || this.itemId;
+    this.status = obj.status || this.status;
+    if (obj.image) {
+      this.shadowRoot.querySelector("img.place-thumbnail").src = obj.image;
```

Comment:
the `||` mean that when obj.address is falsy (for example undefined), then the value will not be replaced. Its a faster way to write:
```js
if (obj.name) {
  this.name = obj.name;
}

```